Document modern forwarding with Paper

mdcfe opened the pull request:
Add details for using modern forwarding with Paper (see PaperMC/Paper#1557).

mdcfe committed:
Document modern forwarding with Paper

Add details for using modern forwarding with Paper
(as per PR PaperMC/Paper#1557).

mdcfe commented:
@astei Now that PaperMC/Paper#1557 has been merged, is there anything that needs altering in this PR?

astei commented on users/player-info-forwarding.rst:
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only recent versions of Paper 1.13.1+ support Velocity's modern forwarding.
```
Include build 377 or higher

mdcfe replied:
Should I put the information about which build supports it here, or under the `Paper` subheading? It's Paper-specific, so I don't think it necessarily belongs immediately below the `Configuring modern forwarding` heading.

astei replied:
Put it in this paragraph.

astei commented on users/player-info-forwarding.rst:
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
+To allow Paper to understand the forwarded player data, in your ``paper.yml``, set
+``settings.velocity-support.enabled`` to ``true`` and ``settings.velocity-support.secret``
+to match the secret in your Velocity config. To run your server in online mode, you can
+optionally set ``settings.velocity-support.online-mode`` to ``true``. Once you're done
```
`Also, you must set settings.velocity-support.online-mode to the online-mode setting in velocity.toml`

mdcfe committed:
Update Paper forwarding paragraph

Update the paragraph on configuring Paper for modern forwarding as per [feedback](https://github.com/VelocityPowered/documentation/pull/2/files#r226721270).

mdcfe committed:
Include more specific Paper version info

astei commented on users/player-info-forwarding.rst:
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only builds 337 and above of Paper 1.13.1+ support Velocity's modern forwarding.
```
https://papermc.io/ci/job/Paper-1.13/377/ is the build that introduced Velocity support.

mdcfe committed:
Fix build number typo
